### ISSUE TITLE
Disappearing inventory

### ISSUE THREAD

**Issue by rxiv** [jakimfett/Werkbench#9]
Whenever crafting something - if you double click to 'gather' items from the crafting grid, everything else disappears.

For instance - placing 3 diamonds and 2 sticks for a diamond pick, and instead of creating the pick, double click on a diamond to retrieve them - the sticks disappear from the crafting grid. if you click where one of the sticks was, they re-appear.

This also happens if you are crafting multiple of something and end up double clicking on the output box - taking a stack of planks and putting them into the grid in a chest pattern, then double click on the chest output box and all the wood in the crafting grid will disappear.

Was wondering if it was an interaction between this and something else, but getting the same thing with no other mods installed (except forge),

**Comment by jakimfett** [comment]
Yeah, the double click issue is one I've had a hard time tracking down. I think it has to do with how I'm syncing the crafting grid with the tileEntity, but sadly my knowledge of the way crafting actually works is...limited at best.

Thanks for reporting this, it's definitely a bug, for now I'd recommend not double clicking.

I'm working on some shift-clicking improvements, but they're kinda on hold while I work on a major rewrite of the GUI stuffs.
